requirements_test_min.txt

```diff
@@ -1,3 +1,2 @@
-astroid==2.5.2
```

Comment:
I might have missed it, but why remove `astroid` here? The idea as I recall was to pin it for the CI tests.

Reply:
Astroid is already defined as a "real" dependency in install_requires.

Reply:
Yeah, but it's only loosely pinned there: `astroid>=2.5.2,<2.7`. That's why I added it to the `requirements` file originally.

Reply:
OK, it makes sense.